packages/create-nx-workspace/src/utils/child-process-utils.ts

```diff
@@ -27,13 +30,21 @@ export function spawnAndWait(command: string, args: string[], cwd: string) {
 }
 
 export function execAndWait(command: string, cwd: string) {
+  output.verbose('Running:', command, 'in', cwd);
   return new Promise<{ code: number; stdout: string }>((res, rej) => {
     exec(
       command,
       { cwd, env: { ...process.env, NX_DAEMON: 'false' } },
       (error, stdout, stderr) => {
         if (error) {
           const logFile = join(cwd, 'error.log');
+          if (isCI()) {
```

Comment:
I don't think we should check this within `create-nx-workspace`. I think people should be able to pass `--verbose` if they are in CI.